clouddriver-azure/src/test/groovy/com/netflix/spinnaker/clouddriver/azure/resources/loadbalancer/deploy/templates/AzureLoadBalancerResourceTemplateSpec.groovy

```diff
@@ -33,12 +33,12 @@ class AzureLoadBalancerResourceTemplateSpec extends Specification {
     expect: template == expectedFullTemplate
   }
 
-  UpsertAzureLoadBalancerDescription createNoRulesDescription(){
-    new UpsertAzureLoadBalancerDescription()
+  AzureLoadBalancerDescription createNoRulesDescription(){
+    new AzureLoadBalancerDescription()
   }
 
-  UpsertAzureLoadBalancerDescription createDescription(){
-    UpsertAzureLoadBalancerDescription description = new UpsertAzureLoadBalancerDescription()
+  AzureLoadBalancerDescription createDescription(){
+    AzureLoadBalancerDescription description = new AzureLoadBalancerDescription()
```

Comment:
If you're going to have the helper method just above, could use it here. Or drop the helper method since it just wraps the no-args constructor anyway.